### PR TITLE
[FIX] Replace lodash.defaultto with koalas

### PR DESCRIPTION
**Pull request by aashna956**
Fixes Issue: https://github.com/gulpjs/vinyl-fs/issues/220

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "graceful-fs": "^4.0.0",
     "gulp-sourcemaps": "^1.5.2",
     "is-valid-glob": "^0.3.0",
+    "koalas": "^1.0.1",
     "lazystream": "^1.0.0",
     "lodash.defaultto": "^4.14.0",
```
Comment (phated) [inline package.json]: Forgot to remove lodash.defaultto dep

Comment (aashna956) [inline package.json]: Fixed!

**Comment by phated** [comment]
👍

**Comment by phated** [comment]
Merged as 55595ae928c0a7b69ddbe953cb8ceb7853e643de

**Pull request closed without merging**